react/week2/NewToDo.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import AddToDo from './AddToDo';
+
+//bootstrap styling
+import { Table, Button, Badge } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+export default function NewToDo() {
+  const [todos, setToDo] = useState([
+    'brush teeth',
+    'go for walk',
+    'take breakfast',
+    'get ready to office',
+    'project meeting',
+    'client meeting',
+  ]);
+  const [mappedToDos, setMap] = useState();
```

Comment:
First, regarding naming: the convention is to name these as `const [someName, setSomeName]` and it's good to keep it consistent, for example `[todos, setTodos]` or `[todoList, setTodoList]` or... you get what I mean. 

Second: you shouldn't be using two hooks for todo items. You should have one hook which would initialize as an array of todo item objects, something like this: 

```
const todos = [
  {
    "id": 1,
    "description": "Get out of bed",
    "completed": true
  },
  {
    "id": 2,
    "description": "Brush teeth",
    "completed": false
  },
  {
    "id": 3,
    "description": "Eat breakfast"
    "completed": false
  }
];

const [todoList, setTodoList] = useState(todos);
```

Afterwards, any action (adding a new todo, marking it as complete or again as incomplete, deleting one) would be achieved using that `setTodoList` function.